babelon/hp-es.babelon.tsv

```diff
@@ -18047,4 +18047,916 @@ en	es	HP:5200426	Delusions of infestation	rdfs:label	Delirios de infestación	OF
+en	es	HP:6000696	Elevated circulating N-carbamyl-beta-aminoisobutyric acid concentration	rdfs:label	Aumento de la concentración de ácido N-carbamil-beta-aminoisobutírico en sangre	OFFICIAL	The concentration of N-carbamyl-beta-aminoisobutyric acid in the blood circulation is above the upper limit of normal.
+en	es	HP:6000697	Low tissue ferrochelatase activity	rdfs:label	Disminución de la actividad tisular de ferroquelatasa	OFFICIAL	Concentration or activity of Ferrochelatase (FECH; EC 4.99.1.1) below the lower limit of normal. FECH enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts. FECH is the terminal enzyme of the heme biosynthetic pathway, and catalyzes the insertion of iron into protoporphyrin to form heme.
+en	es	HP:6000698	Elevated circulating heptacarboxylporphyrin concentration	rdfs:label	Aumento de la concentración de heptacarboxilporfirina en sangre	OFFICIAL	The concentration of heptacarboxylporphyrin in the blood circulation is above the upper limit of normal.
+en	es	HP:6000699	Aggravated by menstruation	rdfs:label	Agravado por la menstruación	OFFICIAL	Applied to a sign or symptom that is more severe during mensutruation.
+en	es	HP:6000700	Reduced breast milk zinc concentration	rdfs:label	Dismunución de la concentración de zinc en la leche materna.	OFFICIAL	The concentration of zink in breast milk is below the lower limit of normal.
+en	es	HP:6000701	Elevated urinary bile acid level	rdfs:label	Aumento del nivel de ácidos biliares en orina	OFFICIAL	The amount of bile acid in the urine, normalized for urine concentration, is above the upper limit of normal.
+en	es	HP:6000702	Elevated urinary glyoxylic acid level	rdfs:label	Aumento del nivel de ácido glioxílico en orina	OFFICIAL	The amount of glyoxylic acid in the urine, normalized for urine concentration, is above the upper limit of normal. The conjugate base of glyoxylic acid is known as glyoxylate.
+en	es	HP:6000703	Elevated circulating 3-hydroxyanthranilic acid concentration	rdfs:label	Aumento de la concentración de ácido 3-hidroxiantranílico en sangre	OFFICIAL	The concentration of 3-hydroxyanthranilic acid in the blood circulation is above the upper limit of normal.
+en	es	HP:6000704	Elevated circulating CXCL9 concentration	rdfs:label	Aumento de la concentración de CXCL9 en sangre	OFFICIAL	The concentration of chemokine (C-X-C motif) ligand 9 in the blood circulation is above the upper limit of normal.;inheres_in;inheres_in_part_of;part_of

```

Comment:
```suggestion
en	es	HP:6000704	Elevated circulating CXCL9 concentration	rdfs:label	Aumento de la concentración de CXCL9 en sangre	OFFICIAL	The concentration of chemokine (C-X-C motif) ligand 9 in the blood circulation is above the upper limit of normal.
```

Seems like a copy/paste error?

Reply:
Yes, you are right. We have corrected this bug and also the 3 typos with mistakes